core/common/src/main/java/alluxio/underfs/UnderFileSystem.java

```diff
@@ -68,7 +68,7 @@ private Cache() {}
        * @param ufsConf the ufs configuration
        * @return the UFS instance
        */
-      UnderFileSystem get(String path, Object ufsConf) {
+      UnderFileSystem get(String path, Map<String, String> ufsConf) {
```

Comment:
Yeah, passing a raw map here seems hard to extend in the future, if we need some other qualifiers from the configuration.